addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -201,7 +211,8 @@
                                         <field name="email_from" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
                                         <label for="reply_to"/>
                                         <div name="reply_to_details">
-                                            <field name="reply_to_mode" widget="radio"
+                                            <field name="reply_to_mode" widget="radio_info"
```

Comment:
Just wondering: is it worth the pain of adding and supporting a new JS field just to add a span ? Can't you achieve this directly in the view itself, using a bit of bootstrap formatting to display it properly ?